engine/gamesys/src/gamesys/wscript

```diff
@@ -46,28 +50,24 @@ def build(bld):
 
     bld.stlib(features = 'cxx ddf',
         includes = '. .. ../../src ../../proto',
-        proto_gen_py = True,
```

Comment:
For waf build artefacts, it'll automatically add a task number, so that the tasks don't collide: "gamesys.cpp" -> "gamesys_0.o", "gamesys_1.o" for different libs etc.

However, for content we generate ourselves, we only need/want one set of test content (e.g. the "__init__.py", so we don't want multiple tasks generating it.
This is what led to the race condition: a lot of tasks generating the same content.

Waf warns about this, if you run with `-v -v -v`(increasing levels of verbosity)